src/javascript/app/pages/user/metatrader/metatrader.ui.js

```diff
@@ -634,17 +680,17 @@ const MetaTraderUI = (() => {
     };
 
     const setDemoTopupStatus = () => {
-        const el_demo_topup_btn  = getElementById('demo_topup_btn');
-        const el_loading         = getElementById('demo_topup_loading');
-        const acc_type           = Client.get('mt5_account');
```

Comment:
Just a general question: previously we added all these spaces because it was a rule defined in eslint or something like that, did we remove this rule before?

Reply:
Stupid IDE :joy: I've fixed it.

Reply:
So, do we still need to keep these spaces? 😂 Because I see your IDE removed it on the other lines as well but eslint passed. Maybe if it's not a requirement anymore, we can just remove it completely to keep the code consistent.

Reply:
@akmal-binary there was no eslint for this, but we want to keep it to maintain readability in binary.

Reply:
Strange, I had a feeling that one time my test failed because of these spaces

Reply:
@akmal-binary that's probably for the `:` separator. for that one we do have an eslint rule